Compile Errors on Windows

This was originally mentioned in #81 and also in https://github.com/ComputationalRadiationPhysics/graybat/issues/41, but since the issue is actually to be fixed in this project, it should be mentioned here.

The following errors/warnings appear when compiling on Windows:

```c++
calc_phi_ase_graybat.cc(168): error C2899: typename cannot be used outside a template declaration
calc_phi_ase_graybat.cc(169): error C2899: typename cannot be used outside a template declaration
calc_phi_ase_graybat.cc(170): error C2899: typename cannot be used outside a template declaration
calc_phi_ase_graybat.cc(171): error C2899: typename cannot be used outside a template declaration
calc_phi_ase_graybat.cc(172): error C2899: typename cannot be used outside a template declaration
calc_phi_ase_graybat.cc(212): warning C4244: 'return' : conversion from 'unsigned int' to 'float', possible loss of data
```

The used build environment was VisualStudio Community 2013.

--- a/src/calc_phi_ase_graybat.cc
+++ b/src/calc_phi_ase_graybat.cc
@@ -165,12 +165,12 @@ float calcPhiAseGrayBat ( const ExperimentParameters &experiment,
      **************************************************************************/
     // Configuration
 
-    typedef typename graybat::communicationPolicy::BMPI CP;
-    typedef typename graybat::graphPolicy::BGL<>        GP;
-    typedef typename graybat::Cage<CP, GP>              Cage;
-    typedef typename Cage::Vertex                       Vertex;
-    typedef typename Cage::Edge                         Edge;
-    
+    typedef graybat::communicationPolicy::BMPI CP;
+    typedef graybat::graphPolicy::BGL<>        GP;
+    typedef graybat::Cage<CP, GP>              Cage;
+    typedef Cage::Vertex                       Vertex;
+    typedef Cage::Edge                         Edge;
+
     // Init
     Cage cage;
     const unsigned nPeers = cage.getPeers().size();

